app/com/yetu/oauth2provider/views/download.scala.html

```diff
@@ -24,11 +24,13 @@
 				<h2>@Messages("setup.download.headline")</h2>
                 <div id="download_available" class="left__content">
                     <p class="left__content-description">@Messages("setup.download.description")</p>
-                    <a id="download_mac1" class="left__content-download" href="https://us-east.manta.joyent.com/yetu/public/downloads/gw-setup-app/mac/yetu-setup.zip"
+                    <a id="download_mac1" class="button left__content-download" href="https://us-east.manta.joyent
+                    .com/yetu/public/downloads/gw-setup-app/mac/yetu-setup.zip"
                         onclick="onClickDownload()">
                         @Messages("setup.download.button.mac")
                     </a>
-                    <a id="download_win1" class="left__content-download" href="https://us-east.manta.joyent.com/yetu/public/downloads/gw-setup-app/windows/yetu-setup.zip"
+                    <a id="download_win1" class="button left__content-download" href="https://us-east.manta.joyent
+                    .com/yetu/public/downloads/gw-setup-app/windows/yetu-setup.zip"
```

Comment:
This line break in the url makes it impossible to the url contains a lot of spaces, thus breaking.
it results in:

```
https://us-east.manta.joyent%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20.com/yetu/public/downloads/gw-setup-app/mac/yetu-setup.zip
```
which, for obvious reasons, fails.

So here we go: a test would catch this mistake. Write one. 

Have a look at https://github.com/yetu/oauth2-provider/blob/master/test/com/yetu/oauth2provider/routes/HealthCheckSpec.scala
It's the easiest test in the project to get started with. 

Have a look around at the other tests around, or check https://www.playframework.com/documentation/2.3.x/ScalaTestingWithScalaTest for some guidance.